### ISSUE TITLE
Closure actions cannot be used as event handlers in Safari

### ISSUE THREAD

**Issue by xtian** [emberjs/ember.js#11604]
In Chrome, the new 1.13 closure actions can be used as event handlers directly on an element, but they throw an exception in Safari:

```handlebars
<div onclick={{action "click"}}></div>

// Clicking on the <div> in Safari throws:
// => SyntaxError: Function statements must have a name. (line 1)
```

Not sure if this officially supported functionality, though. Here are some JSBins:

**1.13:**
http://emberjs.jsbin.com/hamixovace/1/edit?html,js,console,output

**Canary:**
http://emberjs.jsbin.com/hamixovace/2/edit?html,js,console,output

**Comment by mixonic** [comment]
@xtian this behavior is emergent, and not really an intended API. We would like to introduce something we've called "kabob" event listeners in the near term. They would look like:

```
<div on-click={{action "save"}}></div>
```

This needs an RFC and implementation though.

For your immediate case, I suggest using a combination of the element space action helper and the closure action. For example:

```
<div {{action (action "save") on="click"}}></div>
```

Or alternatively (the behavior is slightly different, but similar) just keep using element actions:

```
<div {{action "save" on="click"}}></div>
```

**Comment by xtian** [comment]
Ok, good to know!

**Comment by stefanpenner** [comment]
I believe https://github.com/tildeio/htmlbars/pull/374 fixes this issue.

**Comment by stefanpenner** [comment]
1.13.3 should fix this since  tildeio/htmlbars#374 has been merged
